go/mysql/query.go

```diff
@@ -555,7 +555,7 @@ func (c *Conn) parseComStmtExecute(prepareData map[uint32]*PrepareData, data []b
 		}
 	}
 
-	newParamsBoundFlag, pos, ok := readByte(payload, pos)
+	newParamsBoundFlag, pos, _ := readByte(payload, pos)
```

Comment:
is it ok to ignore the ok here? what if it is false?
The rest look fine.

Reply:
I utilized the ok flag in the latest commit. Should be validating it correctly.